tests/microos/toolbox.pm

```diff
@@ -100,17 +100,8 @@ sub run {
     validate_script_output 'podman ps -a', sub { m/test_name/ };
     assert_script_run 'podman rm test_name';
 
-
-    record_info 'Test', "Rootless toolbox as $user";
```

Comment:
This is still a valid test case, so let's make it condition if we decide to re-add it after fixing. Also let's add a softfailure to https://bugzilla.suse.com/show_bug.cgi?id=1252754.

Something simple like

```
my $bsc1252754 = 1; # rootless toolbox test temporarily disabled until $bsc1252754 is fixed.
if ($bsc1252754) {
record_soft_failure ....
} else {
// the removed code
}
```